src/Algebra/Bundles/Raw.agda

```diff
@@ -27,6 +28,11 @@ record RawSuccessorSet c ℓ : Set (suc (c ⊔ ℓ)) where
     suc#    : Op₁ Carrier
     zero#   : Carrier
 
+  rawSetoid : RawSetoid c ℓ
```

Comment:
I'm surprised that this is here rather than (say) `Relation.Binary.Bundles.Raw`. Sorry if I'm retreading on already-dealt-with topics here, I haven't been following the topic

Reply:
So #2194 *defines* `Relation.Binary.Bundles.Raw` (following a suggestion of yours ;-)), and then this PR is designed to leverage that in order to avoid the duplication of the definition of `_≉_` previously under `Algebra.Bundles`... HTH

Reply:
Ach! I misread this as _defining_ the notion of `RawSetoid`. Serves me right for reviewing PRs overtired and slightly hung over, I guess.

Reply:
Happy New Year! ;-)